src/qibolab/instruments/qblox/cluster_qcm_bb.py

```diff
@@ -128,8 +114,9 @@ def __init__(self, name: str, address: str):
         """
         super().__init__(name, address)
         self.ports: dict = {}
-        self.settings: dict = {}
         self.device: QbloxQrmQcm = None
+        # self.settings: dict = {}
+        self.device = None
```

Comment:
```suggestion
        self.device: QbloxQrmQcm = None
```
`self.device` is defined twice, also dead code can be dropped.